sys/ofed/include/rdma/ib_verbs.h

```diff
@@ -1710,7 +1710,7 @@ struct ib_flow_eth_filter {
 	__be16	ether_type;
 	__be16	vlan_tag;
 	/* Must be last */
-	u8	real_sz[0];
```

Comment:
As the name implies, these are never non-zero length. These are just so the code can calculate the size of the struct without its tail padding. Kind of ugly, but that's how the code works.